.github/workflows/e2e-hardhat.yml.old

```diff
@@ -1,51 +0,0 @@
-name: E2E Hardhat Tests
```

Comment:
Hardhat out, yeah, guess that makes sense.

Not so realistic we'll give this so much attention again in the future.